### DISCUSSION TITLE
Can I get a link to download Github Desktop 64 bit for Windows?

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [community/community#47021] [Other Feature Feedback, Questions, & Ideas]
### I can't reach the Official Home Page of GithubDesktopWIndow . SO I downloaded from https://central.github.com/deployments/desktop/desktop/latest/win32 . And I am not sure it is Safe or not .

**Comment by jge162** [comment]
here is the link -> [GitHub windows desktop](https://desktop.github.com/) just pick the options windows and it will begin the download. 
![Screenshot 2023-02-10 at 7 07 56 PM](https://user-images.githubusercontent.com/31228460/218235962-61570c46-df8e-4e33-b00f-efd69e3d1006.jpeg)

**Reply by Hangell** [reply]
1. Download the GitHub Desktop installation file from the official website: https://desktop.github.com/
2. Save the installation file in the desired folder on your computer.
3. Open PowerShell as administrator. To do this, click the Start button and search for "Windows PowerShell", right-click on the "Windows PowerShell" option and select "Run as administrator".
4. Navigate to the folder where you saved the installation file using the cd commands in PowerShell. For example, if you saved the file in the "Downloads" folder, you can use the command cd Downloads.
5. Run the command .\GitHubDesktopSetup.exe to start the installation. Follow the on-screen instructions to install GitHub Desktop.
6. After installation, you can open GitHub Desktop from the Start menu or the list of installed applications on your computer.
